streamingvisitors/src/vespa/searchvisitor/matching_elements_filler.cpp

```diff
@@ -75,6 +81,24 @@ Matcher::Matcher(FieldIdTSearcherMap& field_searcher_map, const MatchingElements
 
 Matcher::~Matcher() = default;
 
+void
+Matcher::select_multiterm_children(const MatchingElementsFields& fields, const MultiTerm& multi_term)
+{
+    std::optional<std::reference_wrapper<const vespalib::string>> field;
```

Comment:
This might be more straight forward with just a good old fashioned `const vespalib::string* field` with `nullptr` checks